style.css

```diff
@@ -18,26 +16,39 @@ html, body {
     font-family: Verdana, "Comic Sans MS", sans-serif;
 }
 
+#gameScreen{
+    display: none;
+    padding-top: 60px;
+    overflow: hidden;
+}
+
 #gameScreen h1 {
     text-align: center;
     color: #F5F5F5;
     background-color: #874000;
     font-family: "Creepster", sans-serif;
     font-size: 4rem;
     width: 800px;
-    text-align: center;
     margin: 0 auto;
     padding: 20px;
 }
 
+.gameBox {
```

Comment:
this is basically duplicate code for the `.maze` class, feels like you could simplify it

Reply:
removed unnecessary code